File: Scripts/volatile_and_force_compiler_task/exponentiation-operator/javascript/exponentiation-operator.js
function pow(base, exp) {
    if (exp != Math.floor(exp))
        throw "exponent must be an integer";
    if (exp < 0)
        return 1 / pow(base, -exp);
    var ans = 1;
    while (exp > 0) {
        ans *= base;
        exp--;
    }
    return ans;
}

function executeTask(i) {

	return pow(2017+i,12) + pow(19.88+i,12);
}

var r = 0;

for ( var i = 0; i < 2000000000; ++i) {
	r = executeTask(i);
}

File: Scripts/volatile_and_force_compiler_task/exponentiation-operator/ruby/exponentiation-operator.rb
class Numeric
  def pow(m)
    raise TypeError, "exponent must be an integer: #{m}" unless m.is_a? Integer
    Array.new(m, self).reduce(1, :*)
  end
end

  for i in 0..2000000000 
	(2017+i).pow(12) + (19.88+i).pow(12)
  end

